udaykalra/portfolio/src/main/webapp/Contact.html

```diff
@@ -9,10 +9,7 @@
       integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
       crossorigin="anonymous"
     />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
```

Comment:
What is the licensing situation with this asset, and what is it used for (can we avoid it)?

Reply:
That's an import of Bootstrap. The license for it is here which is pretty freeing: https://getbootstrap.com/docs/4.0/about/license/ . I may be wrong but I believe the others' sites also use bootstrap.

Reply:
Correction: That's a deprecated image library and has been removed in next branch. Bootstrap is a different import.